app/helpers/form_helper.rb

```diff
@@ -63,28 +63,34 @@ def mmt_select(options)
     is_multi_select = true if options[:multiple]
 
     select_options = options[:options].clone
-    # restrict options for drop down if for metadata_date
-    select_options.shift(2) if options[:metadata_date]
-
-    disabled_options = []
-
-    if is_multi_select
-      prompt = nil
-      size = 4
-      values = options[:value] || []
-      values.each do |value|
-        if value && invalid_select_option(select_options, value)
-          # handle invalid options for multi_select
-          select_options.unshift value
-          disabled_options << value
-        end
-      end
+    if options[:grouped]
+      select_options = grouped_options_for_select(select_options, options[:value])
```

Comment:
@abbottry what is your opinion?

The code below this adds existing values as disabled options in the select. This is in case the value previously entered isn't part of the controlled keywords. I'm not sure how we would handle this with the options being grouped by Type.

Right now if the existing value isn't found in the short name's it will just display 'Select Short Name'